README.md

```diff
@@ -261,10 +261,22 @@ $response = FigResponseCookies::remove($response, 'theme');
 The `expire` method sets a cookie with an expiry date in the far past. This
 causes the client to remove the cookie.
 
+Note that in order to expire a cookie, you need to configure its `Set-Cookie`
+header just like when you initially wrote the cookie (i.e. same domain/path).
+The easiest way to do this is to re-use the same code for configuring the
+header when setting as well as expiring the cookie:
+
 ```php
 use Dflydev\FigCookies\FigResponseCookies;
+use Dflydev\FigCookies\SetCookie;
+
+$setCookie = SetCookie::create('ba')
+    ->withValue('UQdfdafpJJ23k111m')
+    ->withPath('/')
+    ->withDomain('.example.com')
+;
 
-$response = FigResponseCookies::expire($response, 'session_cookie');
+FigResponseCookies::set($response, $setCookie->expire());
```

Comment:
One alternative I see:

```php
FigResponseCookies::expire($response, $setCookie);
```

Hardly more elegant and slightly harder in terms of BC. Your choice. 😃